src/panels/config/js/condition/logical.tsx

```diff
@@ -0,0 +1,46 @@
+import { h, Component } from "preact";
+
+import Condition from "./index";
+
+export default class LogicalCondition extends Component<any, any> {
+  private _mounted = false;
+  constructor() {
+    super();
+    this.conditionChanged = this.conditionChanged.bind(this);
+  }
+
+  public conditionChanged(conditions) {
+    if (this._mounted) {
```

Comment:
Without this check, things do not work correctly, reproduction steps:
- Create a new automation
- Add a condition and set type to and
- Add two sub-conditions under the and condition, and populate them, for example
![image](https://user-images.githubusercontent.com/14281572/64923060-95572400-d7d6-11e9-8313-d96387978b53.png)
- Change type of the and condition to or:
![image](https://user-images.githubusercontent.com/14281572/64923071-b0c22f00-d7d6-11e9-82e5-3c7c3f8bdb55.png)
- Add a second sub-condition under the or condition - the or condition is reverted back to and:
![image](https://user-images.githubusercontent.com/14281572/64923093-dea77380-d7d6-11e9-9c7e-47c237c76b1e.png)